app/models/piece.rb

```diff
@@ -47,6 +47,7 @@ def is_obstructed?(x, y)
     end
   end
 
+<<<<<<< HEAD
```

Comment:
This file looks like it has some kind of merge-conflict syntax in it. This is what is causing all the errors in CircleCI. This should be removed.